src/map/ai/helpers/pathfind.h

```diff
@@ -41,11 +41,21 @@ class CBaseEntity;
 
 enum PATHFLAG {
   PATHFLAG_NONE			= 0x00,
-  PATHFLAG_RUN			= 0x01, // run twice the speed
-  PATHFLAG_WALLHACK		= 0x02, // run through walls if path is too long
-  PATHFLAG_REVERSE		= 0x04, // reverse the path
-  PATHFLAG_SCRIPT		= 0x08,	// don't overwrite this path before completion (except via another script)
-  PATHFLAG_SLIDE                = 0x10  // Slide to end point if close enough (so no over shoot)
+
+  // run twice the speed
+  PATHFLAG_RUN			= 0x01,
+
+  // run through walls if path is too long
```

Comment:
its the short ones like this (esp on enums), that ppl prefer as inline

Reply:
hmmm but what about the inlines that are wrapping around in full sentences?

Reply:
Those personally drive me bananas. I hate wide lines. I dunno how the other team members feel about them.

Reply:
So I have noticed in other parts of the code that usually `return` statements start on a new line before and after. This is very minor but I am trying to learn this code. How can I improve?